tests/utils/test_utils.go

```diff
@@ -16,11 +16,12 @@ import (
 	clientTests "github.com/jfrog/jfrog-client-go/utils/tests"
 )
 
-func InitSecurityTest(t *testing.T, minVersion string) {
+func InitSecurityTest(t *testing.T, xrayMinVersion, xscMinVersion string) {
 	if !*configTests.TestSecurity {
 		t.Skip("Skipping Security test. To run Security test add the '-test.security=true' option.")
 	}
-	ValidateXrayVersion(t, minVersion)
+	ValidateXrayVersion(t, xrayMinVersion)
+	ValidateXscVersion(t, xscMinVersion)
 }
```

Comment:
check for ValidateXscVersion can be added at the specific test that uses it.
no need to add the check for all tests...

Reply:
there is no separation between xray and XSC tests, there are all security tests, and as far as we get more APIs will approach XSC instead of xray.

Reply:
No need to prepare to that right now, when we will need it, we will change.
This addes more complexity where it is not needed right now